pennylane_qiskit/converter.py

```diff
@@ -584,14 +585,17 @@ def _function(*args, params: dict = None, wires: list = None, **kwargs):
     return _function
 
 
-def load_qasm(qasm_string: str):
+def load_qasm(qasm_string: str, measurements=None):
     """Loads a PennyLane template from a QASM string.
     Args:
         qasm_string (str): the name of the QASM string
+        measurements (None | pennylane.measurements.MeasurementProcess | list[pennylane.measurements.MeasurementProcess]):
+            the PennyLane `measurements <https://docs.pennylane.ai/en/stable/introduction/measurements.html>`_
+            that override the terminal measurements that may be present in the input circuit
```

Comment:
Something here is causing Sphinx build for the docs to fail - I think you need to add a blank line here before the `Returns` section for the docs to render correctly

Reply:
Great, looks like the build succeeds now!